en/docs/learn/design-api/advanced-topics/control-api-visibility-and-subscription-availability-in-developer-portal.md

```diff
@@ -123,6 +123,48 @@ When using the REST API directly, the subscription availability options are avai
     </tr>
     </tbody>
     </table>
+    
+    
+## Control API Document Visibility in Developer Portal
+
+By default, any document associated with an API has the same visibility level of the API. That is, if the API is public, 
+its documentation is also visible to all users (registered and anonymous).
+
+!!! note
+    This feature has been disabled in the default setup.
+     
+To enable other visibility levels to the documentation, go to the  
+`<API-M_HOME>/repository/conf/deployment.toml`, 
+add the following configuration.
+
+``` toml
+[apim.publisher]
+enable_api_doc_visibility = "true"
+```
+
+Document visibility levels can be one of the following options:
+
+- **Same as API visibility:** visible to the same user roles who can see the API. For example, if the API's visibility is public, its documentation is visible to all users.
```

Comment:
these visibility level names do not give the idea of the explained.

Reply:
Changed names in 9234c7c .